upstream-community-operators/postgres-operator/postgres-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,335 @@
+          - list
+          - patch
+          - watch # needed if zalando-postgres-operator account is used for pods as well
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - create
+          - update
+          - delete
+          - get
+        - apiGroups:
+          - ""
+          resources:
+          - nodes
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - persistentvolumeclaims
+          verbs:
+          - delete
+          - get
+          - list
+        - apiGroups:
+          - ""
+          resources:
+          - persistentvolumes
+          verbs:
+          - get
+          - list
+          - update # only for resizing AWS volumes
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - delete
+          - get
+          - list
+          - watch
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          verbs:
+          - create
+          - delete
+          - get
+          - patch
+        - apiGroups:
+          - apps
+          resources:
+          - statefulsets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - policy
+          resources:
+          - poddisruptionbudgets
+          verbs:
+          - create
+          - delete
+          - get
+        - apiGroups:
+          - ""
+          resources:
+          - serviceaccounts
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - "rbac.authorization.k8s.io"
+          resources:
+          - rolebindings
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - "rbac.authorization.k8s.io"
+          resources:
+          - clusterroles
+          verbs:
+          - get
+          - create
+          - delete
+          - patch
+          - list
+          resourceNames:
+          - zalando-postgres-operator
+      deployments:
+      - name: postgres-operator
+        spec:
+          progressDeadlineSeconds: 600
+          replicas: 1
+          revisionHistoryLimit: 2
+          selector:
+            matchLabels:
+              name: postgres-operator
+          strategy:
+            rollingUpdate:
+              maxSurge: 25%
+              maxUnavailable: 25%
+            type: RollingUpdate
+          template:
+            metadata:
+              creationTimestamp: null
+              labels:
+                name: postgres-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCHED_NAMESPACE
+                  value: "*" # listen to all namespaces
```

Comment:
Has this value been tested on a kubernetes cluster? If this operator understands that `*` means that it should watch all namespaces, then this is fine. However, this is the first time I've seen an operator do this, and so I'm skeptical that it will work

Reply:
Nevermind, read the documentation linked above